src/manage_nsfs/nc_lifecycle.js

```diff
@@ -265,7 +271,7 @@ class NCLifecycle {
             if (candidates.delete_candidates?.length > 0) {
                 const expiration = lifecycle_rule.expiration ? this._get_expiration_time(lifecycle_rule.expiration) : 0;
                 const filter_func = this._build_lifecycle_filter({filter: lifecycle_rule.filter, expiration});
-
+                dbg.log0('process_rule: calling delete_multiple_objects, num of objects to be deleted', candidates.delete_candidates.length);
```

Comment:
Are you sure you want to log it in debug level 0?

Reply:
yes this is an important message, why not?

Reply:
Just making sure it is something that we want, it's fine.